Source/Resource/Resources.qrc

```diff
@@ -61,5 +61,89 @@
         <file>Help/Recording/Recording.html</file>
         <file>Help/Recording/record_in_toolbar.jpg</file>
         <file>Help/Recording/record_config.jpg</file>
+        <file alias="Help/qctools.md">../../Docs/qctools.md</file>
```

Comment:
should be automated in https://github.com/bavc/qctools/pull/253/commits/f7d320de4adfaad2362b6e8e0c94df8ed1b7a027, please test